src/encoded/tests/test_upgrade_publication.py

```diff
@@ -55,3 +65,24 @@ def test_publication_upgrade_4_5(upgrader, publication_4):
     value = upgrader.upgrade('publication', publication_4,
                              current_version='4', target_version='5')
     assert value['status'] == 'submitted'
+
+
+def test_publication_upgrade_5_6(upgrader, publication_5):
+    value = upgrader.upgrade('publication', publication_5, current_version='5', target_version='6')
+    assert value['schema_version'] == '6'
+    assert value['status'] == 'in progress'
+    
```

Comment:
Could you check for extra spaces on the empty line 74.   Extra spaces before line endings highlight in red in my text editor so you may not be able to see them as easily.  

I didn't see any other syntax issues in the PR so I'll check the code next.